rs/consensus/src/consensus/validator.rs

```diff
@@ -595,22 +594,75 @@ fn get_notarized_parent(
         .map_err(|_| ValidationFailure::BlockNotFound(parent.clone(), height).into())
 }
 
-/// Collect the min of validated block proposal ranks in the range.
-fn get_min_validated_ranks(
+fn get_disqualified_ranks(
```

Comment:
Should we explain somewhere what a `disqualified` and `qualified` means?
After reading the code it's not clear to me if a block for which we have both a valid notarization and a valid equivocation proof is `qualified` or `disqualified` or both :D

Reply:
Yeah, I clarified what disqualified means. 

```
/// Returns rank map of disqualified ranks in the given range. A rank is
/// considered disqualified at height h, if there exists an equivocation
/// proof for it at that height.
```